src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -738,7 +738,7 @@ type TasksNamesInUI = "External Replication" | "RavenDB ETL" | "SQL ETL" | "Back
 interface sampleCode {
     title: string,
     text: string,
-    html: string
+    html: string | KnockoutComputed<string>;
```

Comment:
why? btw. please use ; at the end of line in interfaces.

Reply:
done -removed computed from here